tests/test_api.py

```diff
@@ -72,7 +72,7 @@ def test_get_pod_for_build(self, osbs):
 
     def test_create_build_with_deprecated_params(self, osbs):
         class MockParser(object):
-            labels = {'Name': 'fedora23/something'}
+            labels = {'Name': 'fedora23/something', 'com.redhat.component': 'fedora_component'}
```

Comment:
Can we make this use `TEST_COMPONENT`, and remove 'component' from the kwargs below?